src/svgui/svgui.ts

```diff
@@ -3,6 +3,8 @@ import * as Backbone from 'backbone';
 import * as _ from 'lodash';
 import * as $ from 'jquery';
 
+const IMAGE_MIN_SIZE: Vector = vector(100, 100);
```

Comment:
Minimum image size should not be a part of `svgui` but provided as parameter from outside.